### DISCUSSION TITLE
For all Radeon 560X (laptop) Owners!

### DISCUSSION THREAD

**Discussion by arkni8** [lshqqytiger/stable-diffusion-webui-directml#462] [Show and tell]
You won!
Well atleast Windows users have won, confirmed. Because I am currently on Windows. I think 560X might be the least powerful GPU currently with the weakest support. No one seem to talk about it, but I know a bunch of people have these gaming laptops with them and all of them had this budget entry level GPU (these competed with 1050Ti from Nvidia, and I know 1050Ti with its CUDA must be doing fairly better than AMD's trembling support lol).

Anyway, I was able to successfully run SD WebUI on Windows with DirectML. (Yes, it was the [lshqqytiger's build](https://github.com/AUTOMATIC1111/stable-diffusion-webui))

For an 512x680 res image, with 20 iterations, I was able to finish renders in just under 1min when running `--opt-sub-quad-attention --medvram --disable-nan-check --skip-torch-cuda-test --use-directml --update-all-extensions --upcast-sampling` argument.

It jumps up to 1.5min or 2min when having `--lowvram` argument instead of `--medvram`.

Realistically, with `medvram`, you can only write like 60+60 tokens (positive prompts + negative prompts) or there about. And the renders are not always great.
With `lowvram`, you can go above 90+90 tokens easily but the renders take about 2 - 2.5 min.

Those arguments I mentioned above are the fastest settings. `upcast-sampling` doesnt seem to do much, but I just have it on there.

If anyone is interested in steps, maybe comment below, I might spend some time writing through the steps.
(PS - ROCm route didn't pan out for me with WSL2, and I didnt try ROCm windows native, partly because RX 560X is not on support list and there is no one talking about it. The couple I read seemed like a lot of time being dumped into it for questionable outcome and since I had other option, I didnt dive into that one.)

**Comment by Ailton54** [comment]
Well, friend, if you're still visiting this forum, I need some help; I simply can't get this to work, and my GPU is exactly that RX 560 8GB model. Thanks in advance.
